### DISCUSSION TITLE
Cannot find module error with Chakra-ui/react 2.10.4

### DISCUSSION THREAD

**Discussion by chaitanya1248** [chakra-ui/chakra-ui#9321] [Q&A] [answered]
React: 18.3.1
React-scripts: 3.2.0

When I try to build the app, I get the following error:

"Failed to compile.

./node_modules/@chakra-ui/react/dist/esm/checkbox/checkbox-context.mjs
Cannot find module: '@chakra-ui/utils/context'. Make sure this package is installed.

You can install this package by running: yarn add @chakra-ui/utils/context."

![image](https://github.com/user-attachments/assets/907511d0-b19a-4b90-af5c-5b2538813282)

Ideally, any child packages are included with the chakra-ui/react package. However, I get this error. Any suggestions?

**Comment by segunadebayo** [comment, marked as answer]
v2 is in maintenance. We are not shipping new v2 work. Stay on 2.8.2 or migrate to v3.
